New: Add the "Inter" font to the font panel

ahankinson opened the pull request:
Thank you for the great software! This is a simple patch to add the 'Inter UI' Google font to the font panel.

passiomatic commented:
Thank you. I guess the font is this: https://fonts.google.com/specimen/Inter?query=inter&preview.text_type=custom#standard-styles  

Note: It looks like it has a bold weight too.  Please add all the available weights.

Also, why are you calling it "Inter UI" ?

ahankinson commented:
Sorry; It (had?) another name, "Inter UI". I guess [they've renamed it now](https://twitter.com/rsms/status/1092277566180777985). 

I've fixed the issue with Bold missing, and removed the "UI" from the font name.

passiomatic commented:
Looks good. I will merge shortly. 

In the long run a more scalable solution is to have a font picker (#21) since people may have different needs regarding fonts. For now I'm just adding fonts which are battle-tested and hi quality. But you know, it is never enough.  :)
